test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -64,6 +65,7 @@ using testing::Unused;
 using namespace std::chrono_literals;
 
 static const uint32_t BufferSize = 100000;
+static const std::string filter_config_name = "scooby.dooby.doo";
```

Comment:
This test here doesn't test the `filter_context_.config_name` is filter name in config. It is jus self-provided and self-assigned. It was ok for hardcoded string before, but here I think we should test with HCM filter config

If config_test or filter_test doesn't have such setup, we can test it in integration test.

Reply:
This can be done in a follow-up PR. I or @jstraceski will do it right after this PR